pkg/metrics/metrics.go

```diff
@@ -262,6 +275,10 @@ func ReportEvictedWorkloads(cqName, reason string) {
 	EvictedWorkloadsTotal.WithLabelValues(cqName, reason).Inc()
 }
 
+func ReportPreemptedWorkloads(preemptorCqName, reason string) {
+	PreemptedWorkloadsTotal.WithLabelValues(preemptorCqName, reason).Inc()
+}
```

Comment:
complement to my comment in `preemption.go`. this is what API could look like
```suggestion
func ReportPreemption(preemptingCqName, preemptingReason, targetCqName string) {
	PreemptedWorkloadsTotal.WithLabelValues(preemptingCqName, preemptingReason).Inc()
	ReportEvictedWorkloads(targetCqName, kueue.WorkloadEvictedByPreemption)
}
```

Reply:
in my opinion `func ReportPreemptedWorkloads(preemptorCqName, reason string) {` is easier to follow.

Reply:
I don't usually like one line functions that could have been inlined. @gabesaba's suggestion adds more value.